src/lib/npm.js

```diff
@@ -67,6 +69,14 @@ module.exports = async function(pkg, info) {
         require_tld: false, // eslint-disable-line camelcase
       }),
     },
+    {
+      type: 'list',
+      name: 'authmethod',
+      message: 'Which authentication method is this npm registry using?',
+      choices: [{name: 'Token based', value: 'token'}, {name: 'Legacy', value: 'legacy'}],
```

Comment:
maybe clarify what `Legacy` means, like `'Legacy (username & password)`'? Maybe it’s clear enough, I can’t tell, as I’ve never run into it myself

Reply:
If we clarify what legacy is we should mention username, password and email, as email is required.

Reply:
Updated